docs/contributing/minikube-dev.md

```diff
@@ -9,14 +9,17 @@ $ kubectl port-forward $tiller_deploy 44134:44134 -n kube-system
 
 In another terminal window:
 ```console
-$ eval $(docker-machine env default)
-$ docker run -dp 5000:5000 registry
-$ draftd start --listen-addr="127.0.0.1:44135" --registry-auth="e30K" --tiller-uri=":44134" --basedomain=k8s.local --local```
+$ registry_pod=$(kubectl get po -n kube-system -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' | grep "registry")
+$ kubectl port-forward $registry_pod 5000:5000 -n kube-system
 ```
 
+In another terminal window:
+```console
+$ eval $(minikube docker-env)
+$ draftd start --listen-addr="127.0.0.1:44135" --registry-auth="e30K" --tiller-uri=":44134" --basedomain=k8s.local --registry.url 10.0.0.68 --local```
```

Comment:
10.0.0.68 should be templatized using `kubectl -n kube-system get svc`. My internal IP address for my registry differs from this example